### DISCUSSION TITLE
Import & Export via API

### DISCUSSION THREAD

**Discussion by blaugrau90** [outline/outline#3624] [Ideas]
It would be very good if there was a possibility to use the export via API for backup purposes. So a backup can be made via the API interface and a ZIP file can be saved accordingly.

**Comment by Jbolt01** [comment]
I believe the Outline API already has functionality for this: https://www.getoutline.com/developers#tag/Collections/paths/~1collections.export_all/post

**Reply by blaugrau90** [reply]
Right. I didn't see that in the API documentation. Thanks a lot ;)
